README.rst

```diff
@@ -38,7 +38,7 @@ Install from pypi:
 
 .. code-block:: shell
 
-    pip install openwisp-users
+    pip3 install openwisp-users
```

Comment:
This is not needed, leave your virtual env figure out which version of python you are using